object_tracker/object_tracker.go

```diff
@@ -190,6 +191,16 @@ func (t *myTracker) run(stream gostream.VideoStream, cancelableCtx context.Conte
 			return
 		default:
 			start := time.Now()
+
+			// Reset if it's the exact time or it's been more than 24 hours since last reset
+			itsBeen := start.Sub(t.lastReset)
+			h, m, s := start.Hour(), start.Minute(), start.Second()
+			if itsBeen >= 24*time.Hour ||
```

Comment:
Make this a variable, not jsut 24 hours. Make it default to 24, but have it be settable by an optional config. And rather than hours, use the golang ParseDuration option to interpret time strings `time.ParseDuration("5h3m")`. 

so the new optional config should default to 24 hours, but be settable with a config like 
```
"reset_duration": "2h5m"
```